deploy/charts/kaudit/templates/kaudit.yaml

```diff
@@ -118,14 +125,19 @@ spec:
           - name: VAULT_AGENT_ADDR
             value: http://localhost:8100
         image: {{.Values.vault.image }}
-        imagePullPolicy: IfNotPresent  
-        securityContext:   
-            allowPrivilegeEscalation: false  
-            capabilities:
-              add:
-                - NET_BIND_SERVICE
-              drop:
-                - ALL                             
+        imagePullPolicy: IfNotPresent
+{{- if eq .Values.k8s.openshift "true" }}
```

Comment:
@Amirilw - if the capabilities can be removed for openshift - it is likely that we can drop this in general

This "allowPrivilegeEscalation: false" should stay for both platforms

Reply:
removed